test/gurka/test_build_admin.cc

```diff
@@ -27,76 +25,129 @@ valhalla::gurka::map BuildPBF(const std::string& workdir) {
         |       |       |   |   |       |
         |       |       |   Z   |       |
         F-------E-------D-------L-------K
+
+
+
+        M-------N-------O
+        |    S-----T    |
+        |    |     |    |
+        |  a-|--b--|-c  |
+        |    |     |    |
+        |    U-----V    |
+        Q-------R-------P
```

Comment:
i added germany with a hole in it an austria inside of it. i was kind of thinking of the jungholz area but thats a more pathological case where the inner austria part touches the main austria part at a single node. so i guess this is a slightly less crazy test